__tests__/app.test.js

```diff
@@ -122,6 +122,95 @@ describe('GET /api/articles/:article_id', () => {
   });
 });
 
+describe('PATCH /api/articles/:article_id', () => {
+  it('200: should increment the vote count on the specified article by the given value and return the article object', () => {
+    return request(app)
+      .patch('/api/articles/1')
+      .send({ inc_votes: 100 })
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+
+        expect(article).toMatchObject({
+          article_id: 1,
+          votes: 200,
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+  it('200: should decrement the vote count on the specified article by the given value and return the article object', () => {
+    return request(app)
+      .patch('/api/articles/1')
+      .send({ inc_votes: -100 })
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+
+        expect(article).toMatchObject({
+          article_id: 1,
+          votes: 0,
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+  it('400: should return a bad request when an invalid article id is provided', () => {
+    return request(app)
+      .patch('/api/articles/invalid-id')
```

Comment:
I would include the .send with a valid object here to ensure you are getting the error for what you are testing! 

I think you've just missed it on this one as you have it in all the others